runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -80,11 +80,19 @@
 		"You can smelt ([0-9+]+|one) more pieces? of iron ore before a ring melts\\.");
 	private static final String RING_OF_FORGING_USED_TEXT = "You retrieve a bar of iron.";
 	private static final String RING_OF_FORGING_BREAK_TEXT = "<col=7f007f>Your Ring of Forging has melted.</col>";
+	private static final Pattern AMULET_OF_CHEMISTRY_CHECK_PATTERN = Pattern.compile(
+		"Your amulet of chemistry has (\\d) charges? left\\."
+	);
+	private static final Pattern AMULET_OF_CHEMISTRY_USED_PATTERN = Pattern.compile(
+		"Your amulet of chemistry helps you create a 4-dose potion. (?:<col=ff0000>)?It has ([0-9]+|one) charges? left."
```

Comment:
The last character should be a literal period, not any character.
```suggestion
		"Your amulet of chemistry helps you create a 4-dose potion. (?:<col=ff0000>)?It has ([0-9]+|one) charges? left\\."
```